assets/styles/_notifications.scss

```diff
@@ -69,6 +69,8 @@ $light-salmon-orange:#FCDDCD;
 $scarlet-red: #B10E1E;
 $pale-violet-red: #ECC3C7;
 
+$govuk-error-colour: 	#d4351c;
```

Comment:
why aren't we using the govuk variables btw? (something I noticed, we barely use govuk classes also?)